plugin/evm/validators_cache_test.go

```diff
@@ -37,7 +37,6 @@ func TestValidatorsCache_ByEpoch(t *testing.T) {
 
 		testValidators := genericValidators(numValidators, numEpochs)
 
-		var calls int
```

Comment:
The calls was removed because it was following the number of times the mock is called.
After the last changes before the merge, the tests were set to not call the mock anymore, but only compare hardcoded values with the result from calling the tested function.
That is why I removed it - because the mock is not called anymore.